src/auto-play-type.js

```diff
@@ -0,0 +1,8 @@
+// @flow
+const AutoPlayType: PKAutoPlayTypes = {
+  ALWAYS: true,
```

Comment:
keep it true, as we don't call false NEVER

Reply:
We could call it OFF but ok, reverted to TRUE :)